e2e/client/playwright/article-versions.spec.ts

```diff
@@ -14,14 +14,21 @@ test.describe('article versions', async () => {
             page.locator(s('article-item=story 2')),
             'Edit',
         );
-        await page.locator(s('authoring', 'field-slugline')).fill('story 2.1');
-        await page.locator(s('authoring-topbar', 'save')).click();
-        await expect(page.locator(s('authoring', 'field-slugline'))).toHaveValue('story 2.1');
+        await sleep(2000);
```

Comment:
Do not use sleep in tests themselves. There is that hack where we do need sleep, but put it to a helper.